lib/basedriver/helpers.js

```diff
@@ -323,7 +322,15 @@ async function unzipApp (zipPath, dstRoot, supportedAppExtensions) {
   try {
     logger.debug(`Unzipping '${zipPath}'`);
     const timer = new timing.Timer().start();
-    const extractionOpts = {};
+    /**
+     * Attempt to use use the system `unzip` (e.g., `/usr/bin/unzip`) due
+     * to the significant performance improvement it provides over the native
+     * JS "unzip" implementation.
+     * @type {import('appium-support/lib/zip').ExtractAllOptions}
+     */
+    const extractionOpts = {
+      useSystemUnzip: !system.isWindows(),
```

Comment:
On Windows we could also use PowerShell for such purpose: https://blog.netwrix.com/2018/11/06/using-powershell-to-create-zip-archives-and-unzip-files/

I could create a PR later

Reply:
this is a good idea, but might not be worth adding unless people want it?